src/pages/EventDetailsPage.tsx

```diff
@@ -1,24 +1,110 @@
 import { ReactElement, useState } from "react";
 import { useParams } from "react-router-dom";
+import Frame from ".././assets/Frame.svg";
+import LinkSVG from ".././assets/Link.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import PinSVG from ".././assets/Pin.svg";
+import TextIconSVG from ".././assets/TextIcon.svg";
 import { fetchEvent } from "../client/client";
 import { Event } from "../util/Types";
 
 export type EventDetailsPageProps = {
-  event?: Event;
+  e?: Event;
 };
 
-const EventDetailsPage = ({ event }: EventDetailsPageProps): ReactElement => {
+//if time is not defined make it all day
+const EventDetailsPage = ({ e }: EventDetailsPageProps): ReactElement => {
   const { id } = useParams();
-  const [eventToDisplay, setEventToDisplay] = useState(event);
+  const [event, setEvent] = useState(e);
 
-  if (!eventToDisplay) {
+  if (!event) {
     fetchEvent(Number(id)).then((e) => {
-      setEventToDisplay(e);
+      setEvent(e);
     });
     return <p>Loading</p>;
-  } else {
-    return <p>{JSON.stringify(eventToDisplay)}</p>;
   }
+
+  const timeString: string = new Date(event.startTime).toLocaleString("en-US", {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    weekday: "long",
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+
+  //variables for displaying the date
+  const startVariables = timeString.split(" ");
+  const month = startVariables[1];
```

Comment:
Instead of splitting the timestring, it would be more readable to use methods like "getMonth()" on the Date
Read: https://www.javatpoint.com/typescript-date-object